nym-vpn-lib/src/lib.rs

```diff
@@ -232,6 +232,7 @@ impl NymVpn<MixnetVpn> {
         exit_point: ExitPoint,
         #[cfg(target_os = "android")] android_context: talpid_types::android::AndroidContext,
         #[cfg(target_os = "ios")] ios_tun_provider: Arc<dyn OSTunProvider>,
+        credential_data_path: Option<PathBuf>
```

Comment:
No need to add it as an argument here, you can just set it in the `sync_run_vpn` function later